File: python/168.Excel-Sheet-Column-Title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        result = ''

        while columnNumber > 0:
            c = chr(((columnNumber - 1) % 26) + 65)
            result = c + result
            columnNumber = (columnNumber - 1) // 26

        return result

File: javascript/168.Excel-Sheet-Column-Title.js
/**
 * @param {number} columnNumber
 * @return {string}
 */
var convertToTitle = function (columnNumber) {
  let result = "";

  while (columnNumber > 0) {
    const c = String.fromCharCode(((columnNumber - 1) % 26) + 65);
    result = c + result;
    columnNumber = Math.floor((columnNumber - 1) / 26);
  }

  return result;
};
